mail/dovecot/Portfile

```diff
@@ -63,16 +64,23 @@ subport dovecot-sieve {
     homepage        https://pigeonhole.dovecot.org/
 
     distname        dovecot-${core_branch}-pigeonhole-${version}
-    master_sites    ${homepage}releases/${core_branch}/
 
     checksums       rmd160  d453de41682ce7c809cabb3c5ebddbe5e31eb804 \
                     sha256  637709a83fb1338c918e5398049f96b7aeb5ae00696794ed1e5a4d4c0ca3f688 \
                     size    1954347
 
     depends_lib-append \
-                    port:dovecot
+                    port:dovecot \
+                    port:gettext \
+                    port:mercurial
```

Comment:
I didn't see any evidence of gettext or mercurial being used, where and why are they needed?

Reply:
Whoops I'll replace this fix.